scripts/consumer_apps_builder.py

```diff
@@ -38,10 +38,16 @@ def checkout_or_update_static_resources_repo():
     os.chdir('../')
 
 
-def build_apks_from_resources(build_type):
+def build_apks_from_resources(build_type, subset_of_apps_to_build):
     for (app_dir_name, sub_dir_list, files_list) in os.walk(PATH_TO_STATIC_RESOURCES_DIR):
         if '.git' not in app_dir_name and app_dir_name != PATH_TO_STATIC_RESOURCES_DIR:
-            build_apk_from_directory_contents(app_dir_name, files_list, build_type)
+            if subset_of_apps_to_build is None or in_list(subset_of_apps_to_build, app_dir_name):
```

Comment:
Sorry I should have explained this, I'll add a comment in the file. This is just a debugging feature for me. This script is set up to pull down the consumer-apps-resources repository and build all of the apps that are in there. But I wanted to have the option of only building some of them while testing, just to make things faster when I was working on a bug in only 1 or 2 of the apps. If I were using this I would just pass in a comma-separated list of directory names as a command line argument to the script invocation:

`python consumer_apps_builder.py d app_Savings,app_WomensHealth`